packages/api/amplify_api/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 0.6.3 (2022-07-21)
+
```

Comment:
question: should this include anything about android version bump? Same question could be asked for other changelogs but seems to fix issue w API.